src/platform/plugins/shared/discover/public/application/main/state_management/discover_data_state_container.test.ts

```diff
@@ -157,10 +148,6 @@ describe('test getDataStateContainer', () => {
       if (hasLoadingMoreStarted && value.fetchStatus === FetchStatus.COMPLETE) {
         expect(resolveDataSourceProfileSpy).not.toHaveBeenCalled();
         expect(value.result).toEqual([...initialRecords, ...moreRecords]);
-        // it uses the same current search session id
```

Comment:
Removed this from the test as we actually don't send the `sessionId` when fetching more anyway: https://github.com/elastic/kibana/blob/c398102c79a48d7f2c26c7f0b73bd16ee9ce9f7c/src/platform/plugins/shared/discover/public/application/main/data_fetching/fetch_documents.ts#L52